### DISCUSSION TITLE
Why another UI framework ? Which difference ? And why not contributing to other ?

### DISCUSSION THREAD

**Discussion by ManUtopiK** [onu-ui/onu-ui#126] [General]
Hi! Your UI framework look awesome and with plenty of components. There are still major components missing, but it looks promising.
Despite, there is already a UI framework using unoCSS and VueUse called [anu](https://github.com/jd-solanki/anu).
I wonder why you started another UI framework instead of contributing to an already existing one ?
Which difference we can expect with onu-ui that anu doesn't have ?
What do you think about collaborate instead of dispatching efforts in different open source projects that use same tools and got same goals ?

**Comment by ManUtopiK** [comment]
Still waiting for a response...

**Comment by yzh990918** [comment]
1. First of all, why did you write this framework? I think the main reason is because I want to provide a contribution to the Vue3 component library of the UnoCSS ecosystem. Everyone loves it so they made such a library together
2. Different from other frameworks, you can refer to our documentation to see https://onu.zyob.top/, which has the following characteristics:
- **Monoreop**: The component library infrastructure is'Monorepo ', and we use the most popular set of tools on the market to build an environment suitable for component development, including some efforts from design packaging architecture, performance optimization, etc
- **Pure UnoCSS writing**, almost all components of the style writing is written using pure UnoCSS, [anu](https://github.com/jd-solanki/anu) is a good UI library, it uses SCSS and UnoCSS to write together, in addition to the above, we also provide UnoCSS-compatible preset methods, so that you can customize your configuration in projects using UnoCSS, including the subsequent we will provide dynamic configuration theme API, further perfect compatibility with UnoCSS, of course, because of the use of UnoCSS, our style file size is almost reduced several times, HMR startup speed is also fast.
- **setup script**, all our components are written in Vue3 setup script, not use `TSX`
- **dark mode  & i18n**, we have customized a set of color variables for dark mode, and support international language switching

Of course, the above points are still my temporary thoughts. Of course, there are very few components now. We will continue to write complex and commonly used components in the future. This is already planned, so the production environment of this component library is not recommended. I will add some of what I can think of when I release the stable version later. Finally, thank you for your attention 🙏 .

**Reply by ManUtopiK** [reply]
Thanks for taking time to respond.
Despite your long answer, the main question was « I wonder why you started another UI framework instead of contributing to an already existing one ? »
I'm always boring about open source disperstion of efforts...

[Anu](https://github.com/jd-solanki/anu) is also built in monorepo, use pure unocss (and soon compatible with tailwindcss), use Vue3 script setup and got dark mode out of the box. It's also really configurable. Onu looks great, but I don't see any advantages with onu over anu.
